toolchain/stage2/declare_toolchains.bzl

```diff
@@ -0,0 +1,43 @@
+load("@rules_cc//cc/toolchains:toolchain.bzl", "cc_toolchain")
+load("//platforms:common.bzl", _supported_targets = "SUPPORTED_TARGETS", _supported_execs = "SUPPORTED_EXECS")
+load("//toolchain:selects.bzl", "platform_cc_tool_map")
+
+def declare_toolchains():
```

Comment:
I'm up for referencing the rules_cc folks explanation as to why we have to declare N targets for M execs !